fedorable.sh

```
@@ -402,223 +241,160 @@ task_autoremove() {
 
 task_clean_temp_files() {
     print_header "Temporary File Cleanup (Age > $TEMP_FILE_AGE_DAYS days)"
-    log_msg "Searching for old temporary files in /tmp and /var/tmp..."
 
-    local found_tmp=0 found_var_tmp=0
     if [[ $DRY_RUN -eq 1 ]]; then
-        found_tmp=$(find /tmp -type f -atime +"$TEMP_FILE_AGE_DAYS" -print 2>/dev/null | wc -l)
-        found_var_tmp=$(find /var/tmp -type f -atime +"$TEMP_FILE_AGE_DAYS" -print 2>/dev/null | wc -l)
-        log_msg "[DRY RUN] Would delete $found_tmp file(s) from /tmp."
-        log_msg "[DRY RUN] Would delete $found_var_tmp file(s) from /var/tmp."
+        local found_tmp=$(find /tmp -type f -atime +"$TEMP_FILE_AGE_DAYS" -print 2>/dev/null | wc -l)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)